### PR TITLE
refactor: drop usage of _.partial

### PR DESCRIPTION
**Pull request by JustinBeckwith**
refactor: drop usage of _.partial

### CODE REVIEW

File: packages/grpc-native-core/src/client.js [hunk 1]
```diff
@@ -939,9 +939,11 @@ exports.makeClientConstructor = function(methods, serviceName,
       throw new Error('Method names cannot start with $');
     }
     var method_type = common.getMethodType(attrs);
-    var method_func = _.partial(requester_funcs[method_type], attrs.path,
-                                attrs.requestSerialize,
-                                attrs.responseDeserialize);
+    var method_func = function() {
+      return requester_funcs[method_type].apply(this, arguments.concat([
+        attrs.path, attrs.requestSerialize, attrs.responseDeserialize
```
Comment (murgatroid99) [inline packages/grpc-native-core/src/client.js]: I think this is putting those arguments at the end of the final list, instead of at the beginning.

**Comment by murgatroid99** [comment]
OK, you didn't actually change the thing I was pointing out. You're still concatenating the preset arguments at the end of the arguments list, but they should go at the beginning.

**Comment by JustinBeckwith** [comment]
🤦‍♂️

**Comment by murgatroid99** [comment]
I think slicing the arguments list is now redundant.

**Comment by murgatroid99** [comment]
The Linux build failure was a Kokoro error, and there's no way it was impacted by this change.

**Pull request merged**